### PR TITLE
daemon: /v2/system-recovery-keys remove API

### PR DESCRIPTION
**Pull request by pedronis**

### CODE REVIEW

**Comment by pedronis** [comment]
based on #11714

File: daemon/api_system_recovery_keys.go [hunk 1]
```diff
@@ -43,3 +47,39 @@ func getSystemRecoveryKeys(c *Command, r *http.Request, user *auth.UserState) Re
...
+	if err := decoder.Decode(&postData); err != nil {
+		return BadRequest("cannot decode recovery keys action data from request body: %v", err)
+	}
+	if decoder.More() {
+		return BadRequest("spurious content after recovery keys action")
+	}
+	switch postData.Action {
+	case "":
+		return BadRequest("missing recovery keys action")
+	default:
```
Comment (bboozzoo) [inline daemon/api_system_recovery_keys.go:71]: nit: even though it doesn't affect the outcome, i'd put this last

**Review by bboozzoo — APPROVED** [review]
The last 2 commits LGTM

**Review by stolowski — APPROVED** [review]
Looks good, thank you

**Pull request merged**